tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -37,7 +37,8 @@ describe("E2E Testing for taqueria typechecker and simulator tasks of the tezos-
         expect(accounts.stdout).toContain("bob")
     });
 
-    test('Verify that taqueria flextesa plugin will return "Already running." if sandbox has started" if user tries to call start sandbox twice', async () => {
+    // TODO: Come back to!
```

Comment:
@mweichert what is there to come back to here?

Reply:
I'll try reverting this change. If you recall last week, I had some issues with tests running locally but not in CICD. This was an attempt to get tests running in CI/CD.